test/functional/test_configuration_change.py

```diff
@@ -185,6 +186,32 @@ def test_qemu_boot_from_disk_img(self):
                         password="root", timeout=120)
         ssh.close()
 
+    @helper.qemu_version(">=2.10")
+    def test_auto_add_nvme_serial(self):
+        self.conf["compute"]["storage_backend"] = [{
+            "type": "nvme",
+            "cmb_size": 256,
+            "drives": [{"size": 8}]
+        }]
+        self.conf["compute"]["qemu_bin"] = "/home/infrasim/isilon/idic/qemu/x86_64-softmmu/qemu-system-x86_64"
```

Comment:
This is temporary solution to hard-code where compatible qemu binary is placed. May be add a comment of "fixme".

Reply:
Good catch! This config is a temporary local test config, to verify a good behavior on qemu 2.10, which should be removed.